src/server.jsx

```diff
@@ -174,6 +174,23 @@ server
 
           if (context.url) {
             res.redirect(context.url);
+          } else if (context.error_code) {
+            res.set({
+              'Cache-Control': 'no-cache',
+            });
+
+            res.status(context.error_code).send(
+              `<!doctype html>
+                ${renderToString(
+                  <Html
+                    extractor={extractor}
+                    markup={markup}
+                    store={store}
+                    extractScripts={false}
```

Comment:
@tiberiuichim In development environment, the error pages, with this PR, do not load the styles that they should have. This change corrects this (I am not sure if this has or doesn't have other negative implications):

```suggestion
                    extractScripts={process.env.NODE_ENV !== 'production'}
```

HTH